test/Syntax/round_trip_parse_gen.swift

```diff
@@ -143,6 +143,7 @@ typealias H = () rethrows -> ()
 typealias I = (A & B<C>) -> C & D
 typealias J = inout @autoclosure () -> Int
 typealias K = (@invalidAttr Int, inout Int, __shared Int, __owned Int) -> ()
+typealias L = A<B, C>
```

Comment:
What is the meaning of this added line of code?

Reply:
@xwu I asked to add it because we didn't have test cases for TypeSyntax with multiple generic arguments.

@pi1024e You probably need to update `test/Syntax/Outputs/round_trip_parse_gen.swift.withkind`. Please test it locally.